### DISCUSSION TITLE
Using this app with Mihon at the same time?

### DISCUSSION THREAD

**Discussion by RowanSkie** [aniyomiorg/aniyomi#1837] [General]
Should I use this with Mihon at the same time? Can I migrate everything I have on Mihon though if not recommended?

**Comment by equalDelight** [comment]
I think you should use both because Mihon is still in active development while Aniyomi barely had an update. Use Aniyomi to view anime only.

Aniyomi's backup also isn't compatible with Mihon, so if you use Aniyomi to read and it somehow gets an error, you'll be manually migrating from Aniyomi to Mihon; dreadful, lmao.

**Reply by Flitch300** [reply]
Just attempted this after seeing this why did I only realize this now 😭 (at least I noticed before it got TOO deep).

**Reply by falconsena-creator** [reply]
(Português Brasil).
Você pode sim transferir suas coisas do aniyome para o Mihon. Porém você deve salvar algo no Mihon, fazer backup.
Depois faça backup do aniyome.
Depois você entre na pasta do aniyome e substitua algumas coisas, exemplo: nome da pasta. E funcionará normalmente. Comigo funcionou.


(English Short)
You can do it, but you should back up both and replace the name "aniyome" with "Mihon". Then it will work normally.

**Comment by Memo560** [comment]
Hello, I am an Arabic manga reader. I have a problem. All the sources in the Arabic language do not work. What should I do?

**Reply by AbduAlZahra313** [reply]
That's because they have been taken down by an Asshole Korean Manhwa Company, like how GManga fell.

**Reply by Memo560** [reply]
Thank you
